.env.example

```
@@ -0,0 +1,5 @@
+
+# Slack config keys
+SLACK_API_TOKEN=${SLACK_OAuth_Acces_Token}
+SLACK_BOT_TOKEN=${SLACK_Bot_User_OAuth_Access_Token}
+SLACK_CHANNEL=${CHANNEL_ID}
```

```suggestion
SLACK_API_TOKEN=${SLACK_API_TOKEN}
SLACK_BOT_TOKEN=${SLACK_BOT_TOKEN}
SLACK_CHANNEL_ID=${SLACK_CHANNEL_ID}
```
Lets be specific with our environment variable names and use same key for value so that it will be easier for us to substitute the value in CI/CD or other places.